common/character_interactions/01_republic_interactions.txt

```diff
@@ -56,7 +56,7 @@
 					is_magus_trigger = yes
 				}
 			}
-			multiply = 0
```

Comment:
so factor here ? because i had problems with this the factor few lines above saying its unknown token

Reply:
Oops, you are correct. This is a script value, not a modifier field, so multiply is the correct thing to use here.